.storybook/FileDownload/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import styles from './styles.css';
+import file from './file.svg';
+
+class FileDownload extends Component {
```

Comment:
Actually, there's no need for this to be a class. You could just use a function like we do for our other simple components.

Reply:
I'll have a look and restructure it.

Reply:
Done.